assignment2/src/components/Question2.jsx

```diff
@@ -0,0 +1,129 @@
+import React, {useState} from 'react'
+
+
+function Question2() {
+      const [fName, setFName] = useState("")
```

Comment:
Please use single userDetails object as state to store all user details. This will reduce number of re-rendering .